### ISSUE TITLE
create cheat sheets for Bash-Snippets in cheat.sheets

### ISSUE THREAD

**Issue by chubin** [alexanderepstein/Bash-Snippets#64]
Create cheat sheets for all commands/scripts in Bash-snippets in cheat.sheets (https://github.com/chubin/cheat.sheets), so users could find the best Bash-Snippets usage examples using a Bash-Snippet tool (cheat) only (without man, google, stackoverflow etc.).

~~~~
$ cheat ytview 
...
$ cheat stocks
...
~~~~

You can create cheat sheets directly here (in this ticket) and then I add them to cheat.sh (not the best option), but it would be better if you could add them to the `cheat.sheets` repository directly (the bes option for now).

The third (and probably the best option in long run) if you add a `cheat.sheets` directory to your repository and it will be bound to cheat.sh (so you'll update your cheat sheets on your own and they will be still available to all cheat.sh users; read more on it in the cheat.sh README).

To summarize:

Bash-Snippets cheat sheets page are good because:

1) Bash-Snippets users can read its docs using only Bash-Snippets tools;
2) It will help other users of cheat.sh who are not aware of Bash-Snippets to discover it.
(if someone runs some seqrch query like say: `cheat.sh/~youtube/i` it will finds the snippets or to be more concrete the `stocks` command).
3) To start, create the cheat sheets and add them to cheat.sheets (and later migrate them to this repository).

**Comment by alexanderepstein** [comment]
Will look into adding this sometime soon, good idea.

**Comment by alexanderepstein** [comment]
Created cheatsheets <a href="https://github.com/chubin/cheat.sheets/pull/3">here</a>.

**Comment by chubin** [comment]
Thank you! I've merged it, it should be avaiable now.
Please try:
```
cheat movies
cheat short
```

Could you please also (1) give a short one line description
for each tool?

```
cloudup
crypt
currency
geo
movies
qrify
short
siteciphers
stocks
taste
todo
weather
ytview
```

and also (2) we need the parent-snippet for the whole tools "Bash-Snippets"
with a small description how to install and use the snippets (because otherwise you don't know from the cheat sheets where you can get these tools).

Can you create she Bash-Snippets cheat sheet?

**Comment by alexanderepstein** [comment]
Added description to each cheatsheet and created a bash-snippets cheatsheet here https://github.com/chubin/cheat.sheets/pull/4

**Comment by chubin** [comment]
I've updated the cheat sheets according to the cheat sheets syntax rules + renamed `bash-snippets` to `Bash-Snippets`.
Now you can list all bash-snippets available without installing them (perfect for the new users, that want to get an overview of the Bash-Snippets from the console). Just issue:

```
curl cheat.sh/~Bash-Snippets
```

Consider adding this command to the main README.md of Bash-Snippets.
People should know how they get the actual Bash-Snippets list without installing anything
and even without a browser!

(in the future we will add some additional info to the output: last version, last pubiished feature etc.)
+ we will move the cheat.sheets to this repository so you'll always can keep them up to date.

Question:

Does `cheat` (from Bash-Snippets) support tab completion? (cheat.sh does).
If not you should add it. And after that you can add bash completion to many of the Bash Snippets scripts (if they don't have any at the moment).

Write me if you need an example how you can do it.

**Comment by alexanderepstein** [comment]
Would you be able to submit a pull request adding the cheat sheets to this repository? I just want to make sure it is done correctly. 

At the moment none of the tools support bash completion, I don't personally use this too often which is why it's not there. I have never really taken the time to learn it.
